src/javascripts/components/mushroomList/mushroomList.js

```diff
@@ -1,3 +1,6 @@
+import firebase from 'firebase/app';
+import 'firebase/storage';
```

Comment:
we need to import the firebase storage lib so we can interact with our storage out on firebase